scipy/spatial/ckdtree.pyx

```diff
@@ -825,9 +850,10 @@ cdef class cKDTree:
             nearest points are further than ``r / (1 + eps)``, and branches are
             added in bulk if their furthest points are nearer than
             ``r * (1 + eps)``.
-        n_jobs : int, optional
+        workers : int, optional
             Number of jobs to schedule for parallel processing. If -1 is given
             all processors are used. Default: 1.
+            Note: This argument was renamed from "n_jobs" in SciPy version 1.6.0
```

Comment:
Should way say when it should be removed? Maybe 1.8.0?

Reply:
Added here and in the warning message.